### PR TITLE
DL-6021 Auth Upgrade

### PR DESCRIPTION
**Pull request by Sir-Iyke**
DL-6021 Auth Upgrade

**New feature**

Update bootstrap to 5.2.0 

## Checklist

 - [x]  I've made every effort to commit high quality, clean code and I have executed relevant static analyses to be sure
 - [ ]  I've included appropriate tests with any code I've added (Unit, Integration, Acceptance etc.)
 - [x]  I've executed the acceptance test pack locally to ensure there are no functional regressions
 - [x]  I've added my code using logical, atomic commits, squashing as appropriate - including the JIRA issue number in the commit message
 - [x]  I've run a dependency check to ensure all dependencies are up to date

### CODE REVIEW

File: project/plugins.sbt [hunk 1]
```diff
@@ -1,4 +1,4 @@
-resolvers += "HMRC-open-artefacts-maven" at "https://open.artefacts.tax.service.gov.uk/maven2"
+resolvers += MavenRepository("HMRC-open-artefacts-maven2", "https://open.artefacts.tax.service.gov.uk/maven2")
 resolvers += Resolver.url("HMRC-open-artefacts-ivy", url("https://open.artefacts.tax.service.gov.uk/ivy2"))(Resolver.ivyStylePatterns)
 
 addSbtPlugin("uk.gov.hmrc" % "sbt-auto-build" % "2.15.0")
```
Comment (paulanderson1234) [inline project/plugins.sbt:4]: Latest version ? are we applying ?

Comment (Sir-Iyke) [reply project/plugins.sbt:4]: Are we not supposed to?

Comment (Sir-Iyke) [reply project/plugins.sbt]: Yes, i've added it now

**Review by benryan1790 — COMMENTED** [review]
Need to look again at ETMPConnector. The cGET method should look like this:

`@inline def cGET[A](url: String)(implicit rds: HttpReads[A], hc: HeaderCarrier): Future[A] = {
    val future: Future[A] = http.GET[A](url, headers = Seq("Authorization" -> urlHeaderAuthorization, "Environemnt" -> urlHeaderEnvironment)(rds, hc), ec = ExecutionContext.global)
    future.foreach {
      case e: Exception => loggingUtils.err(s"get request failed: url=$url\ne=$e\n")
    }
    future
  }`

**Pull request merged**